core/src/main/java/com/klaytn/caver/Caver.java

```diff
@@ -67,11 +68,16 @@
      */
     public IPFS ipfs;
 
-    /*
+    /**
      * The AccountWrapper instance
      */
     public AccountWrapper account;
 
+    /**
+     * The ContractWrapper instance
+     */
+    public ContractWrapper contract;
+
     public Caver() {
         this(new HttpService(DEFAULT_URL));
```

Comment:
Please add a function description.

Reply:
@sirano11 
Thanks for your review :)

I added comment here [commit](https://github.com/klaytn/caver-java/pull/221/commits/6050503e450f8f9d35c6f4f31dc5547004c7e3b3).

Big thanks.